Purchases/RCPurchases+Protected.h

```diff
@@ -16,11 +16,14 @@ NS_ASSUME_NONNULL_BEGIN
 - (instancetype _Nullable)initWithAppUserID:(NSString * _Nullable)appUserID
                              requestFetcher:(RCStoreKitRequestFetcher *)requestFetcher
                              receiptFetcher:(RCReceiptFetcher *)receiptFetcher
+                         attributionFetcher:(RCAttributionFetcher *)attributionFetcher
                                     backend:(RCBackend *)backend
                             storeKitWrapper:(RCStoreKitWrapper *)storeKitWrapper
                          notificationCenter:(NSNotificationCenter *)notificationCenter
                                userDefaults:(NSUserDefaults *)userDefaults;
 
++ (void)setDefaultInstance:(RCPurchases * _Nullable)instance;
```

Comment:
`RCPurchases` -> `instancetype`

Reply:
Do you mean changing RCPurchases with instancetype? Can you do that for parameters?